.github/workflows/ci.yml

```diff
@@ -79,7 +79,8 @@ jobs:
 
   # Build github-metrics-aggregator image and push to artifact registry
   build:
-    runs-on: 'ubuntu-latest'
+    runs-on:
+      group: 'Beefy' # group with a 4-core and 8-core runner for faster builds
```

Comment:
@pdewilde - will it randomly choose one? I created the _group_ because that's the only way to restrict the runners to a specific repo. I was thinking you would use either the 4-core or 8-core machine label directly.

Reply:
Aah, I assumed you intended for me to use the group. I'm not sure what the logic is. I did one test and saw it corresponded to the 8-core machine and was hoping it would use the more powerful machine first.

I'll have to take a look

Reply:
https://github.com/actions/runner/issues/1665

Looks like there is a long-existing ask but not movement on such a prioritization feature in a year. (That is for Self-Hosted rather than Large runners, but I would imagine the behavior is similar).

Probably best to just slap on the 8-core machine label directly. Probably no need to have a 4-core for now in the Beefy group as most of the other jobs run in pretty similar times.